tests/single-node/requirements.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+yum install iptables-services -y
```

Comment:
Is this really a requirements or a bug in kubespray (or MetalK8s) ?

Reply:
It is required on the `tests/single-node/test.sh` script otherwise the following lines fail:
https://github.com/scality/metal-k8s/blob/15e6cf826a684c2fb3da933000fe1bf74acb4d71/tests/single-node/test.sh#L22

If you think that the proper fix would be to deactivate the service if it exists I can change it as well. I don't think it's required on kubespray or metalk8s AFAIK

Reply:
If whatever image this is now running on doesn't come with the `iptables` service, there's no need to install it if we want to disable it for the test environment anyway.

The `single-node` test script should be updated to
- Check whether the `iptables` service exists
- If it exists, stop it

Reply:
Agree
Will fix it in the `single-node` script